.circleci/config.yml

```diff
@@ -363,6 +363,10 @@ jobs:
           test_targets: "posixtest"
   test-core0:
     executor: bionic
+    # Temporarily set EMCC_READ_METADATA to compare to ensure that the python
+    # can marches precisely the output wasm-emscripten-finalize.
+    environment:
+      EMCC_READ_METADATA: "compare"
```

Comment:
Is this meant for landing?

Reply:
Yes, I'd like to run at least one test suite in this mode for as long as we have two different ways to do this.   So that we can be sure they are kept in sync.